src/js/sliding/listeners.js

```diff
@@ -270,72 +264,24 @@ export const setUpListeners = (
     menuButton.addEventListener('click', handleMenuButtonClick);
     toggles.forEach(toggle => {
       const handler = e => handleToggle(e, toggle);
-      toggleHandlerFunctions.push(handler);
       toggle.addEventListener('click', handler);
     });
     dropdownNavLists.forEach(dropdown => {
       const handler = e => handleDropdownNavList(e, dropdown);
-      dropdownNavListsHandlers.push(handler);
       dropdown.children[1].addEventListener('keydown', handler);
     });
     // eslint-disable-next-line no-undef
     document.querySelectorAll('.js-back-button').forEach(backButton => {
       const handler = e => handleGoBackOneLevel(e, backButton);
-      goBackOneLevelHandlers.push(handler);
       backButton.addEventListener('click', handler);
     });
   };
 
-  const removeListeners = () => {
```

Comment:
With so much of the code being removed. Is it not needed? There was a reason why it was there (I suppose)?